app-devices/yubioath-flutter/autobuild/build

```diff
@@ -0,0 +1,5 @@
+abinfo "Building yubioath-flutter"
+fvm flutter build linux
+
+abinfo "Creating PKGDIR ..."
```

Comment:
I see. This is still not needed - you should really just move merge `beyond` into this script - `beyond` is really meant for post-build adjustments after build templates.

I will clarify this in the Styling Manual.